chromium_src/chrome/browser/notifications/notification_platform_bridge_android.cc

```diff
@@ -3,8 +3,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#include "brave/build/android/jni_headers/BraveNotificationPlatformBridge_jni.h"
 #include "chrome/browser/notifications/notification_platform_bridge_android.h"
+
+#include "brave/build/android/jni_headers/BraveNotificationPlatformBridge_jni.h"
+#include "third_party/skia/include/core/SkBitmap.h"
```

Comment:
why is skia added here?

Reply:
otherwise there is a build error, for the same reason they moved header here https://source.chromium.org/chromium/chromium/src/+/refs/tags/125.0.6422.9:chrome/browser/notifications/notification_platform_bridge_android.cc;drc=22c523349416ff59df49b160f89e7c0ee0acb662;l=44
in this change https://source.chromium.org/chromium/chromium/src/+/22c523349416ff59df49b160f89e7c0ee0acb662

Reply:
The main reason, is because now we have `@JniType("SkBitmap") ` param in `NotificationPlatformBridge.displayNotification` and `SkBitmap` has to be explicitly added before `NotificationPlatformBridge_jni.h`

Reply:
It seems like we're missing something else then because these jni files should already contain the headers the need.

Reply:
strange, ok